webservices/resources/legal.py

```diff
@@ -9,8 +9,15 @@
 from webargs import fields
 from pyelasticsearch import ElasticSearch
 from webservices.env import env
+from flask import request
 
 
+if env.get_credential('WRITE_AUTHORIZED_TOKENS'):
```

Comment:
How about `env.get_credential('WRITE_AUTHORIZED_TOKENS', '12345, ...')`? Also, do the default tokens have any meaning? Do we need them? Seems like it would be easy to forget to add this credential to a user-provided service and open up the write api to everybody who takes the time to look at the source.